config/boards/README.md

```diff
@@ -4,6 +4,9 @@
 | BOARD_NAME | welcome text and hostname |
 | BOARDFAMILY | [sun8i, sun7i, rockchip64](../sources) |
 | BOOTCONFIG | name of u-boot config |
+| BOOTCONFIG_DEFAULT | name of u-boot config for default branch |
+| BOOTCONFIG_DEV | name of u-boot config for dev branch |
+| BOOTCONFIG_NEXT | name of u-boot config for next branch |
```

Comment:
There is no `next` branch anymore. It has been replaced with `current`. Any remainings are leftovers.

Reply:
I've included it just because it was used in `helios4.conf` file. Should I remove only `BOOTCONFIG_NEXT` or all `BOOTCONFIG_${BRANCH}` descriptions?

Reply:
You can leave it, maybe marking as deprecated is the proper way.